src/Runner/Runner.php

```diff
@@ -134,8 +134,16 @@ public function fix()
             Tokens::clearCache();
 
             if ($fixInfo) {
-                $name = $this->directory->getRelativePathTo($file);
-                $changed[$name] = $fixInfo;
+                $path = $file->getPathname();
+
+                /* workaround for Symfony/Finder setting doubled slash for root folders in
+                SplFileInfo. See `RecursiveDirectoryIterator::current()`. */
+                if (substr($path, 0, 2) === '//') {
```

Comment:
I do not think so as cannot really happen: PHP cannot handle microsoft's special paths starting with `\\` (for network resources) and cannot imagine any other usage of `\\` - if you know any plz let me know. Sorry, cannot bring any out of my head at the moment.